### PR TITLE
Make retry logging more readable

### PR DESCRIPTION
**Pull request by fregante**
The previous logging was easy to handle but hard to follow and noisy. I think this will help debugging

## Before

Screenshot 3

## After

Screenshot 4

### CODE REVIEW

File: source/sender.ts [hunk 1]
```diff
@@ -58,25 +62,25 @@ function manageConnection(
...
   });
 }
 
 async function manageMessage(
   type: string,
   target: AnyTarget,
-  sendMessage: () => Promise<unknown>
+  sendMessage: (attempt: number) => Promise<unknown>
 ): Promise<unknown> {
   const response = await pRetry(
```
Comment (fregante) [inline source/sender.ts:81]: Side note: p-retry’s stack traces are not great because they're handled via `setTimeout`. I might replace it with a plain `while(retries--) await sleep()` loop.

**Review by grahamlangford — APPROVED** [review]
